src/components/deck-gl-wrapper/deck-gl-wrapper.tsx

```diff
@@ -621,8 +625,8 @@ export const DeckGlWrapper = ({
     if (layer.token) {
       loadOptions.i3s.token = layer.token;
     }
-    return new Tile3DLayer({
-      id: `tile-layer-${layer.id}-draco-${useDracoGeometry}-compressed-textures-${useCompressedTextures}--colors-by-attribute-${colorsByAttribute?.attributeName}--colors-by-attribute-mode-${colorsByAttribute?.mode}--${loadNumber}`,
+    return new CustomTile3DLayer({
+      id: `tile-layer-${layer.id}-draco-${useDracoGeometry}-compressed-textures-${useCompressedTextures}--${loadNumber}` as string,
```

Comment:
`as string` doesn't look necessary

Reply:
Removed, there was a error when I did this change last time, now works fine